src/julia-parser.scm

```diff
@@ -15,7 +15,7 @@
 (define prec-conditional '(?))
 (define prec-arrow       (append!
                           '(-- -->)
```

Comment:
Is there a reason, we don't have these with dots? If we have dotted versions of `<--` and `<-->`, it probably makes sense to have those as well.

Reply:
That's part of the `-->` issue I referenced above.

Reply:
Ah, right! Didn't read Stefan's comment closely enough.